PalasoUIWindowsForms/Reporting/MemoryManagement.cs

```diff
@@ -72,25 +74,37 @@ public static bool CheckMemory(bool minor, string eventDescription, bool okToBot
 					totalVirtualMemory = ((availableMemory > twoGB ? twoGB : availableMemory) / 1024).ToString("N0");
 				}
 #endif
+			long virtualMemory64;
+			long peakVirtualMemory64;
+			long peakWorkingSet64;
```

Comment:
I would have put the comments for these at their declaration (and declaration at assignment, since they're not used outside the using block)